axe_selenium_python/tests/test_axe.py

```diff
@@ -4,45 +4,51 @@
 
 from os import path
 import pytest
+from axe_selenium_python import Axe
 
 
 @pytest.mark.nondestructive
-def test_get_rules(axe):
+def test_get_rules(selenium):
     """Assert number of rule tests matches number of available rules."""
+    axe = Axe(selenium)
     axe.inject()
     rules = axe.get_rules()
-    assert len(rules) == 58, len(rules)
+    assert len(rules) == 60, len(rules)
 
 
 @pytest.mark.nondestructive
-def test_execute(axe):
+def test_execute(selenium):
     """Run axe against base_url and verify JSON output."""
+    axe = Axe(selenium)
     axe.inject()
     data = axe.execute()
     assert data is not None, data
```

Comment:
Can you look for something specific here? Checking for it to be ```not None``` isn't really a test.